lms/djangoapps/canvas_integration/client.py

```diff
@@ -181,7 +181,7 @@ def create_assignment_payload(subsection_block):
                 # string representation.
                 else subsection_block.fields["due"].astimezone(pytz.UTC).isoformat()
             ),
-            "submission_types": ["online_upload"],
+            "submission_types": ["none"],
```

Comment:
How did you determine this value, "none" ?

Reply:
If these are the relevant docs, it seems like we are limited in our options here: 

https://canvas.instructure.com/doc/api/submissions.html#method.submissions.create

Reply:
@pdpinch I got it from here
```
const override = mockOverride({
    submissionTypes: [
      'none',
      'external_tool',
      'online_upload',
      'on_paper',
      'online_text_entry',
      'online_url'
    ]
  })
```
https://github.com/instructure/canvas-lms/blob/45961e84b828b8f56e487de29e405c6ca97a1631/app/jsx/assignments_2/teacher/components/Overrides/__tests__/OverrideSubmissionTypes.test.js#L24

Reply:
I had requested to grant me access to the course page on the ticket couple and one day ago, to verify that value but did not get any response from your side.

Reply:
@pdpinch that value could be verified only from the page source that what's I need access and requesting that on the ticket for last few days.

Reply:
<img width="899" alt="Screenshot 2020-12-10 at 19 21 06" src="https://user-images.githubusercontent.com/4043989/101784213-08a42e80-3b1d-11eb-8d66-25db1dba772b.png">